docs/Platform-Architecture.md

```diff
@@ -22,11 +22,13 @@ To learn more about the `manager_params` and `browser_params` have a look at [Co
 
 ## Watchdogs
 In OpenWPM we have a so called watchdog that tries to ensure two things.
-- `process_watchdog` 
-    * It is an optional parameter that can be passed to the `TaskManager` to create another thread that kills off `GeckoDriver` (or `Xvfb`) instances that haven't been spawned by OpenWPM. (GeckoDriver is used by Selenium to control Firefox and Xvfb a "virtual display" so we simulate having graphics when running on a server)
+- `process_watchdog`
+    * It is part of default manager_params. It is set to false by default which can manually be set to true.
+    * It is used to create another thread that kills off `GeckoDriver` (or `Xvfb`) instances that haven't been spawned by OpenWPM. (GeckoDriver is used by Selenium to control Firefox and Xvfb a "virtual display" so we simulate having graphics when running on a server).
```

Comment:
```suggestion
    * It is used to create another thread that kills off `GeckoDriver` (or `Xvfb`) instances that aren't currently controlled by OpenWPM. (GeckoDriver is used by Selenium to control Firefox and Xvfb a "virtual display" so we simulate having graphics when running on a server).
```
Usually these instances have been spawned by OpenWPM but we forgot to close them when handling an exception or similiar